assets/js/localization.js

```diff
@@ -52,7 +53,12 @@ $(document).ready(function () {
         iso639CodesInverse[language] = code;
     });
 
-    var possibleItems = {'translation': getPairs, 'generation': getGenerators, 'analyzation': getAnalyzers};
+    var possibleItems = {
+        'translation': getPairs,
+        'generation': getGenerators,
+        'analyzation': getAnalyzers,
+        'spellchecker': getSpellers
```

Comment:
Is it just me or is something off, shouldn't it be 'spell checking'? Otherwise, 'translation' would be 'translator', etc.

Reply:
(and in other analagous places)